prodtests/full-system-test/aggregator-workflow.sh

```diff
@@ -110,9 +110,9 @@ if workflow_has_parameter CALIB_PROXIES; then
         # expand FLPs; TPC uses from 001 to 145, but 145 is reserved for SAC
         for flp in $(seq -f "%03g" 1 144); do
           FLP_ADDRESS="tcp://alicr1-flp-ib${flp}:${TPC_IDC_FLP_PORT}"
```

Comment:
Can it be that the host names are not correct? Should they not be of the form `alio2-cr1-flp101.cern.ch`?

Reply:
@davidrohr told me to use this address. Maybe he can comment?

Reply:
The "o2" was missing, but the "ip" should be correct. Marking this as solved, but we still see the issue below.